cmd/create.go

```diff
@@ -408,10 +408,11 @@ func createHost(provider, name, region, zone, projectID, userData, inletsPort st
 			Additional: map[string]string{},
 		}, nil
 	} else if provider == "civo" {
+		const civoUbuntu2004ID = "d927ad2f-5073-4ed6-b2eb-b8e61aef29a8"
```

Comment:
`civo` is redundant given the scope, but I am OK with this. Thank you 👍

Reply:
done